helpers/mathjax.js

```diff
@@ -61,6 +61,16 @@ class HtmlBlockMathRenderer {
 			elm.style.height = '0.5rem';
 		});
 
+		// WIRIS outputs bad syntax for empty sub/superscripts, but changing these on write
+		// can be risky. Instead, change them on render to minimize potential damage if this
+		// impacts legitimate usages.
+		if (context.replaceNoneSubSuperScripts) {
+			elem.querySelectorAll('math mmultiscripts > none').forEach(elm => {
```

Comment:
I don't believe these can nest, as the expected syntax involves adding more children to the `mmultiscripts` element if you want more sub/super-scripts. I also want this selector to be as specific as possible to avoid collateral damage.